tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -73,8 +73,9 @@ class WskBasicUsageTests
         Wsk.exists(wsk.usePythonCLI)
     }
 
-    it should "show help and usage info" in {
-        val stdout = wsk.cli(Seq("-h")).stdout
+    it should "show help and usage info using the default language" in {
+        val env = Map("LANG" -> "de_DE")
```

Comment:
This is a modification to an original test case.  The orginal test was to confirm the existence of the help and usage.  Since the test case is hard coded with English, the test uses the default locale of English.  This test will not change when non-English translations are ready; it's an English only test.

With the additional of a non-English locale, the test case is now testing that English help and usage appears when a non-English locale is specified.  This is a short term test and will need updating when non-English translations are available.  IMHO, this test is separate from the original test intent.